examples/step-15/step-15.cc

```diff
@@ -683,7 +683,7 @@ namespace Step15
         ++refinement_cycle;
         std::cout << std::endl;
       }
-    while (last_residual_norm > 1e-3);
+    while (last_residual_norm > 1e-2);
```

Comment:
Have you checked whether that needs updating the results section?

Reply:
Yes, and the results are fine: they are already truncated at a residual of about `0.09`.